### DISCUSSION TITLE
Frontend Team Meeting on April 21

### DISCUSSION THREAD

**Discussion by meminciftci** [bounswe/bounswe2024group11#204] [General]
What are your available time slots for today's (April 21) frontend meeting?

@umitcan07 @tenacke @meminciftci @hks1444 @ArdaVural

**Comment by hks1444** [comment]
I have another meeting at 9 so anything before 9 is okay for me.

**Comment by meminciftci** [comment]
I will be available around 8 so after 8 is OK for me.

**Reply by umitcan07** [reply]
8 is ok for both of us.
Me and Emre
